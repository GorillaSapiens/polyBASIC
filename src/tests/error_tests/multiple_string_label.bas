fred a := 12
fred b := 5
