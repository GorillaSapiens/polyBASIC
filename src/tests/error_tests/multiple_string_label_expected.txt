ERROR: label ❮fred❯ at 2:6 already defined 1:6
